pages/common/tsort.md

```diff
@@ -7,3 +7,7 @@
 - Perform a topological sort consistent with a partial sort per line of input separated by blanks:
 
 `tsort {{path/to/file}}`
+
+- Perform a topological sort consistent on strings:
+
+`echo "UI Backend\nBackend Database\nDocs UI"|tsort`
```

Comment:
```suggestion
`echo "{{UI Backend\nBackend Database\nDocs UI}}"|tsort`
```
Thanks for your contribution, since this is a placeholder, I would suggest adding `{{}}` to be highlighted in our clients.

Reply:
Right, thanks for the suggestion :)